src/index.js

```diff
@@ -25,6 +29,7 @@ module.exports = {
       return [
         ...fullCorvidTypes.BASE,
         ...fullCorvidTypes.PAGES,
+        ...getJswTypeDeclarations(),
```

Comment:
should we unite common thing like jsw, dependencies, other ? under the base context or some other common function ?

Reply:
United jsw & json as commons.
Made a unified API for all and imported them via index file